lib/galaxy/jobs/__init__.py

```diff
@@ -1743,7 +1743,14 @@ def _change_ownership( self, username, gid ):
         job = self.get_job()
         # FIXME: hardcoded path
         external_chown_script = self.get_destination_configuration("external_chown_script", None)
-        cmd = [ '/usr/bin/sudo', '-E', external_chown_script, self.working_directory, username, str( gid ) ]
+
+        cmd = [ '/usr/bin/sudo', '-E', ]
+        for v in ["PATH", "LD_LIBRARY_PATH", "PKG_CONFIG_PATH"]:
```

Comment:
The security policy of out cluster has the env_reset option enabled (see man 5 sudoers) which in particular resets path. This seems to be the default on CentOS 6.8. In addition the LD_LIBRARY_PATH and PKG_CONFIG_PATH are reset. I'm not sure if PKG_CONFIG_PATH is actually needed - I just added everything that was removed. 

I think this modification is likely not of interest for many (maybe only me). The question would be how to proceed here. A possibility for myself would be to patch my local copy. Btw. what would be the "git way" to do such a thing?